src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+import { DEFAULT_IMG } from '../../utils';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAddMovieBtn: (newMovie: Movie) => void;
+  thisMovieExist: boolean
```

Comment:
it's better to pass callback like `checkMovieExist` which will check is movie in your movies and return boolean value

Reply:
Instead of one short state word you offer me to create a separete function which one return a boolean, it dosent seems needed